packages/generator/src/generation/context.ts

```diff
@@ -0,0 +1,9 @@
+import { TableType, UiFramework } from '../definition/context-types'
+import { Entity } from "./entity"
+
+export default interface GenerationContext {
+    useFormatter: boolean
+    tableType: TableType
+    uiFramework: UiFramework
+    entity: Entity
```

Comment:
I think we should rename property name `entity` to express the semantics/intention. I propose `inputEntity: Entity`